src/components/InputField/styles.ts

```diff
@@ -0,0 +1,36 @@
+import styled from '@emotion/styled';
+import { InputProps } from 'src/components/inputField/types';
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  gap: 8px;
+  width: 335px;
+  height: 72px;
+`;
+
+export const Label = styled.label`
+  font-size: 16px;
+  color: #000;
+`;
+
+export const Input = styled.input<InputProps>`
+  width: 100%;
+  padding: 8px;
+  border-radius: 8px;
+  border-width: 1px;
+  border: solid;
+  border-color: ${props => (props.hasError ? 'red' : '#ccc')};
+  outline: none;
+
+  &:focus {
+    border-color: #000;
```

Comment:
1.3.2

Reply:
fixed colors to props.theme.colors.black in this commit https://github.com/ZenBit-Tech/faq_team_fe/pull/1/commits/6946b1fe06bc34499ad0193d2c5396d51a17d410